assets/product-images-slider.js

```diff
@@ -0,0 +1,62 @@
+const thumbnailSlides = document.querySelector('[data-glide-thumbnail-slides]').dataset.glideThumbnailSlides
+const thumbnailsPerView = thumbnailSlides ? thumbnailSlides : 0
+
+const glideOptions = {
+  image: {
+    type: 'carousel',
+    startAt: 0,
+    perView: 1,
+    focusAt: 'center',
+    gap: 0,
+    dragThreshold: 10,
+    peek: 40,
+  },
+  thumbnail: {
+    type: 'carousel',
+    startAt: 0,
+    perView: thumbnailsPerView,
+    focusAt: thumbnailsPerView % 2 === 0 ? (thumbnailsPerView / 2) - 1 : 'center'
+  }
+}
+
+const createGlideInstance = (element, optionsObject) => {
+  return new Glide(element, optionsObject)
```

Comment:
I believe that since this is an arrow function, you do not need the return, you can just remove the curly brackets and put `new Glide(element, optionsObject)` right after the arrow